plugins/woocommerce-admin/client/marketplace/components/load-more-button/load-more-button.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { speak } from '@wordpress/a11y';
+import { queueRecordEvent } from '@woocommerce/tracks';
+
+interface LoadMoreProps {
+	onLoadMore: () => void;
+	isBusy: boolean;
+	disabled: boolean;
+}
+
+export default function LoadMoreButton( props: LoadMoreProps ) {
+	const { onLoadMore, isBusy, disabled } = props;
+	function handleClick() {
+		queueRecordEvent( 'marketplace_load_more_button_clicked', {} );
+		onLoadMore();
+	}
+
+	if ( isBusy ) {
+		speak( __( 'Loading more products', 'woocommerce' ) );
+	}
+
+	return (
+		<Button
+			className="woocommerce-marketplace__load-more"
+			variant={ 'secondary' }
+			onClick={ handleClick }
+			isBusy={ isBusy }
+			disabled={ disabled }
+		>
+			{ __( 'Load more', 'woocommerce' ) }
```

Comment:
This is fine for now, athough we chatted about infinite loading on the code chat call today.

Maybe we can create an issue about it to see if it's worth pursuing? May improve UX and especially discoverability (h/t @andfinally)